data/test/scenarios/test_add_or_sub_in_leadership.cfg

```diff
@@ -0,0 +1,153 @@
+# This unit test is used for verify the work of abilities used like weapons, but also of special_id_active attribute.
```

Comment:
It's good to test both things, but it would be even better to have 2 simpler separate tests instead of a single combined test.

For the leadership test, it seems cleaner to remove the filters. Just keep the `[attacks]`, `[damage]` and `[chance_to_hit]` to make the test deterministic.

For testing `special_id_active`, it should be on something like `[attacks]` or `[damage]` where the test would reliably fail. When it's on `[chance_to_hit]`, there's a chance the RNG will make the test pass even if the `special_id_active` code is broken.

Reply:
I is done, (i hope!)

Reply:
The comment still says that this tests the `special_id_active` attribute, but I think that has already been removed. I think removing it was good, but the comment needs to be updated too.